features/bookmarks.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Cite and Email Bookmarks
+  As a logged in user,
+  I would like to bookmark certain records
+  and be able to share them 
+  by citing and emailing them.
+
+@bm
```

Comment:
@barnabyalter , thanks. I forgot to remove it when I pushed it up. It was just to do quicker testing, I'll make sure to remove it once I'm done. Thanks!